internal/endtoend/endtoend_test.go

```diff
@@ -44,6 +44,30 @@ func TestExamples(t *testing.T) {
 	}
 }
 
+func BenchmarkExamples(b *testing.B) {
+	examples, err := filepath.Abs(filepath.Join("..", "..", "examples"))
+	if err != nil {
+		b.Fatal(err)
+	}
+	files, err := ioutil.ReadDir(examples)
+	if err != nil {
+		b.Fatal(err)
+	}
+	for _, replay := range files {
+		if !replay.IsDir() {
+			continue
+		}
+		tc := replay.Name()
+		b.Run(tc, func(b *testing.B) {
+			path := filepath.Join(examples, tc)
+			var stderr bytes.Buffer
```

Comment:
You probably want to reset stderr between runs otherwise it will grow without bound.